src/extension.ts

```diff
@@ -67,18 +67,33 @@ export async function activate(context: vscode.ExtensionContext) {
         const resourceFetcher = new DefaultResourceFetcher()
         const regionProvider = new DefaultRegionProvider(context, resourceFetcher)
 
+        ext.awsContextCommands = new DefaultAWSContextCommands(awsContext, awsContextTrees, regionProvider)
+        ext.sdkClientBuilder = new DefaultAWSClientBuilder(awsContext)
+        ext.toolkitClientBuilder = new DefaultToolkitClientBuilder()
+
         // check to see if current user is valid
         const currentProfile = awsContext.getCredentialProfileName()
         if (currentProfile) {
-             const successfulLogin = await UserCredentialsUtils.addUserDataToContext(currentProfile, awsContext)
-             if (!successfulLogin) {
+            try {
+                const successfulLogin = await UserCredentialsUtils.addUserDataToContext(currentProfile, awsContext)
+                if (!successfulLogin) {
+                    await UserCredentialsUtils.removeUserDataFromContext(awsContext)
+                }
+            } catch (err) {
+                const error = err as Error
+                vscode.window.showErrorMessage(
```

Comment:
Error looks like this: 
![credentialserr](https://user-images.githubusercontent.com/29374703/58344395-658ca680-7e0a-11e9-895e-68a7df7b7e68.PNG)